Makefile

```diff
@@ -471,3 +471,8 @@ all-images: jupyter-minimal-ubi9-python-$(RELEASE_PYTHON_VERSION) \
 	rocm-jupyter-pytorch-ubi9-python-$(RELEASE_PYTHON_VERSION) \
 	rocm-runtime-pytorch-ubi9-python-$(RELEASE_PYTHON_VERSION) \
 	rocm-runtime-tensorflow-ubi9-python-$(RELEASE_PYTHON_VERSION)
+
+# This is used primarly for konflux_generate_component_build_pipelines.py to we know the build release version
+.PHONY: print-release
+print-release:
+	@echo "$(RELEASE)"
```

Comment:
```
❯ gmake -nps | grep '^RELEASE '
+ which podman
+ pwd
RELEASE = 2024b
```

already sort of does what is needed, and we were pulling out stuff from Makefile with `-nps` before. If you really want custom target, add it to the list on top as well

https://github.com/opendatahub-io/notebooks/blob/4c99b519571fa9b3099b56818006c48404fcf84c/Makefile#L105

Reply:
I was also considering that approach too... but having a separate target seemed a bit more clean solution to me. That's why I went this way.